contracts/Shortfall/Shortfall.sol

```diff
@@ -236,10 +281,18 @@ contract Shortfall is Ownable2StepUpgradeable, AccessControlledV8, ReentrancyGua
         Auction storage auction = auctions[comptroller];
 
         require(_isStarted(auction), "no on-going auction");
-        require(
-            block.number > auction.highestBidBlock + nextBidderBlockLimit && auction.highestBidder != address(0),
-            "waiting for next bidder. cannot close auction"
-        );
+
+        if (Toggle == Mode.BLOCK_NUMBER) {
+            require(
+                block.number > auction.highestBidBlock + nextBidderBlockLimit && auction.highestBidder != address(0),
```

Comment:
We could use the overridden function `getBlockNumberOrTimestamp()` (see above). So, we wouldn't need the if/else statement